pyfolio/deprecate.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for marking deprecated functions."""
+# Copyright 2016 Quantopian, Inc.
```

Comment:
I don't know the answer to that. I think we were told to do it like that from our OSS expert advisor.